fortuna/src/Main.css

```diff
@@ -7,34 +7,39 @@ body, html {
     background: #000921;
 }
 
-h1 {
-    padding: 5px 5px 5px 5px;
+/* Navbar Styling */

```

Comment:
Stuff specific to the navbar should be in its own file imo. This file should be for generic stuff that anyone might want to use when then need a button, div et cetera.

Reply:
Gonna fix this issue in my next pull.